src/main/java/nl/workingtalent/wtacademy/book/Book.java

```diff
@@ -23,7 +23,7 @@ public class Book {
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private long id;
 
-	@Column(length = 450)
+	@Column(length = 450, nullable = false)
 	private String title;
 
 	private String description;
```

Comment:
Zouden description, publishingDate en isbn ook geen length moeten hebben? bijvoorbeeld een ISBN is altijd 13 digits. Dat zou potentiële fouten kunnen voorkomen bij het aanmaken.